CCDB/test/testCcdbApi.cxx

```diff
@@ -164,6 +164,10 @@ BOOST_AUTO_TEST_CASE(store_test, *utf::precondition(if_reachable()))
   auto h1 = new TH1F("object1", "object1", 100, 0, 99);
   h1->FillRandom("gaus", 10000);
   f.api.store(h1, "Test/Detector", f.metadata, -1, -1, true);
+
+  auto h2 = new TH1F("object2", "object2", 100, 0, 99);
+  h1->FillRandom("gaus", 10000);
+  f.api.store(h1, "Test/Detector", f.metadata, -1, -1, true);
```

Comment:
should this be `h2` in the last 2 lines (or what is the purpose of h2?)?

Reply:
you are absolutely correct. My copy pasting capabilities are decreasing.